scripts/serverSetup.md

```diff
@@ -0,0 +1,66 @@
+# Getting started with YesWorkflow Web Components
+
+**Please ensure that JDK 10 and Maven 3+ are installed on your machine before continuing and confirm that JDK 10 is your default JAVA_HOME**
```

Comment:
maven 3 is purely for developing yesworkflow prototypes cli, we don't want to include this in instructions on how to deploy a server. a user would consume the .jar file or native c++ executable to use the CLI